examples/routes-rest.js

```diff
@@ -17,13 +17,9 @@
 //
 // To run this integrations use:
 //
-//     kamel run --name=withrest --dependency=camel-undertow examples/routes-rest.js
+//     kamel run --name=withrest
```

Comment:
Maybe this does not work ;)

Reply:
It seems not to be working with the Undertow dependency, as two servers get started. I'm catching up though :)

Reply:
Ah damn, I get your point :D

Reply:
It should be better now :D